static/app/views/issueDetails/groupMerged/index.tsx

```diff
@@ -135,7 +135,7 @@ class GroupMergedView extends Component<Props, State> {
         <Layout.Main fullWidth>
           <HeaderWrapper>
             <Title>
-              {tct('Merged fingerprints with latest event [count]', {
+              {tct('Fingerprints included in this issue [count]', {
                 count: <QueryCount count={fingerprintsWithLatestEvent.length} />,
               })}
```

Comment:
super nit tct isn't really helping here
```suggestion
              {t('Fingerprints included in this issue [count]')} <QueryCount count={fingerprintsWithLatestEvent.length} />
```

Reply:
Ah, right, because `tct` is a singular/plural handler, right? Must have been leftover from some previous iteration of the header.

Will make the change, thanks.